public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -127,21 +127,23 @@ <h5 class="col-xs-12 box-title">
           </div>
         </div>
         <div class="row">
-          <div class="col-md-4 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
-          <div class="col-md-4 grid-header flex-xs-middle"> Number of particpants </div>
-          <div class="col-md-3 grid-header flex-xs-middle info-text"> <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon> What is SNOMED? </div>
+          <div class="col-md-3 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
+          <div class="col-md-2 grid-header flex-xs-middle"> Concept Id </div>
+          <div class="col-md-4 grid-header flex-xs-middle"> Number of participants </div>
+          <div class="col-md-2 grid-header flex-xs-middle info-text"> <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon> What is SNOMED? </div>
           <div class="col-md-1 grid-header flex-xs-middle info-text"> View graphs </div>
 
         </div>
         <div class="row expandable-grid-row"
              *ngFor="let r of items.concat(standardConcepts), let i = index"
              [ngClass]="{'expanded-grid-row': r.expanded}" >
-          <div class="col-md-4 flex-xs-middle" innerHtml="{{i+1}}. {{r.conceptName | highlightSearch : prevSearchText}}"> </div>
+          <div class="col-md-3 flex-xs-middle" innerHtml="{{i+1}}. {{r.conceptName | highlightSearch : prevSearchText}}"> </div>
+          <div class="col-md-2 flex-xs-middle" innerHtml="{{r.conceptId}}"> </div>
           <div class="col-md-4 flex-xs-middle">
             {{r.countValue | number }} out of {{totalParticipants | number}} participants
 
           </div>
-          <div class="col-md-3 flex-xs-middle">
+          <div class="col-md-2 flex-xs-middle">
```

Comment:
md-4 for this. or md-3 .. But this column is wider than the num participants and the concept_id ... That is the gist.

Reply:
Done. The text of number of participants was big like "172,340 out of 946,240 participants" and concept code with vocabulary id was little less, so i thought this would be better. Anyway, I made them both 3,3. Thanks.

Reply:
I'm surprised they didn't offer text fix for that column. It looks the worst.  Dot this : Put the total number of participants in the column header like : Number of Participants  <br/> <b> 199,999 total </b>  
Then just show the count in the column